psamm/datasource/sbml.py

```diff
@@ -1020,4 +1035,5 @@ def write_model(self, file, model):
         self._add_gene_list(model_tag, gene_ids)
 
         tree = ET.ElementTree(root)
+        self._indent(root)
```

Comment:
I would like to only apply this if an option `--pretty` is provided to the `sbmlexport` command.